src/main/java/com/openlattice/edm/types/processors/UpdatePropertyTypeMetadataProcessor.java

```diff
@@ -56,6 +55,9 @@ public Object process( Entry<UUID, PropertyType> entry ) {
             if ( update.getPii().isPresent() ) {
                 pt.setPii( update.getPii().get() );
             }
+            if ( update.getIndexType().isPresent() ) {
+                pt.setPostgresIndexType( update.getIndexType().get() );
```

Comment:
is there logic somewhere to actually update the index type in postgres?

Reply:
Uhh now that you mention it, I'm not really seeing anything to do that. The update logic seems pretty constrained

Reply:
I think we should probably include that piece with the ability to change the indexType field on property types, otherwise we could end up in a weird state where the indexType field doesn't actually reflect the index type in postgres

Reply:
although actually we might already be in that state...

Reply:
👍  will do!

Reply:
Actually let's scratch that and make a ticket for it

Reply:
https://jira.openlattice.com/browse/LATTICE-1983